installers/ansible/roles/pgo-operator/templates/deployment.json.j2

```diff
@@ -77,6 +77,10 @@
                                     }
                                 }
                             },
+                            {
+                                "name": "PGO_DISABLE_POSTGRESCLUSTER",
+                                "value": "true"
+                            },
```

Comment:
Do we need this?

Reply:
This will simply prevent a `pgcluster`-based Deployment from also trying to run the `postgrescluster` controllers, which require their own specific resources (e.g. RBAC) that is managed and handled separately from this installer (specifically in the Kustomize installer for the `postgrescluster`-based Deployment).